src/components/GameItem.js

```diff
@@ -4,23 +4,50 @@ import ItemIcons from '../ItemIcons.js';
 import PropTypes from 'prop-types';
 
 class GameItem extends Component {
-  propTypes = {
+  static propTypes = {
     height: PropTypes.number.isRequired,
     layer: PropTypes.number.isRequired,
   }
 
+  constructor(props) {
+    super(props);
+    this.state = {
+      clickedOn: false,
+    };
+  }
+
+  onClickChange = (event) => {
+    this.setState({
+      clickedOn: true,
+    });
+
+    const pointsOfItems = this.props.type === "litter" ? 1 : 0;
+    // function passed on from parent (App.js) through props
+    //Game sends info about points by passing in data to send to parent
+    this.props.onClickForPoints(pointsOfItems);
```

Comment:
You should also make sure that you can't get points for clicking on the same piece of litter multiple times.